src/openg2p_spar_mapper_api/services/mapper.py

```diff
@@ -21,6 +21,7 @@
     ResolveScope,
```

Comment:
@vineela-afk Can you update the `openg2p_g2pconnect_common_lib` to include the below imports inside the `__init__.py`?

`ResolveScope`
`ResolveStatusReasonCode`
`SingleUpdateRequest`